CalcualtorLogic.js

```diff
@@ -18,16 +18,88 @@ class CalculatorLogic {
+        resultDiv.classList.add("result");
+        resultDiv.id = "result";
+        resultDiv.appendChild(
+            document.createTextNode(this.currentResult)
+        );
+        this.resultElement = resultDiv;
+        wrapperDiv.appendChild(resultDiv);
+        const row1 = this.getButtonsRow(["7", "8", "9"]);
+        const row2 = this.getButtonsRow(["4", "5", "6"]);
+        const row3 = this.getButtonsRow(["1", "2", "3"]);
+        const row4 = this.getOperatorsRow(["*", "+", "-"]);
+        wrapperDiv.appendChild(row1);
+        wrapperDiv.appendChild(row2);
+        wrapperDiv.appendChild(row3);
+        wrapperDiv.appendChild(row4);
+        const div4 = document.createElement("div");
+        const button0 = document.createElement("button");
+        button0.dataset.number = "0";
+        button0.classList.add("button");
+        const button0Text = document.createTextNode("0");
+        button0.appendChild(button0Text);
+        const periodButton = document.createElement("button");
+        periodButton.classList.add("button");
+        const periodButtonText = document.createTextNode(".");
+        periodButton.appendChild(periodButtonText);
+        const negativeValueButton = document.createElement("button");
+        negativeValueButton.classList.add("button");
+        const negativeValueButtonText = document.createTextNode("±");
+        negativeValueButton.appendChild(negativeValueButtonText);
+        div4.appendChild(button0);
+        div4.appendChild(periodButton);
+        div4.appendChild(negativeValueButton);
+        wrapperDiv.appendChild(div4);
+        this.hostElem.appendChild(wrapperDiv);
+    };
+    getButtonsRow(buttons) {
+        const div = document.createElement("div");
+        for (let button of buttons) {
+            var buttonElement = document.createElement("button");
+            buttonElement.dataset.number = button;
+            buttonElement.classList.add("button");
+            const buttonElementText = document.createTextNode(button);
+            buttonElement.appendChild(buttonElementText);
+            div.appendChild(buttonElement);
+            buttonElement.addEventListener("click", this.onButtonNumberClick);
+        }
+        return div;
+    }
+    getOperatorsRow(buttons) {
+        const div = document.createElement("div");
+        for (let button of buttons) {
+            var buttonElement = document.createElement("button");
+            buttonElement.dataset.operator = button;
+            buttonElement.classList.add("button");
+            const buttonElementText = document.createTextNode(button);
+            buttonElement.appendChild(buttonElementText);
+            div.appendChild(buttonElement);
+        }
+        return div
     };
-    render() {};
+    onButtonNumberClick(event) {
+        const currentResultAndclickedButton = this.currentResult + event.target.dataset.number;
+        console.log(parseInt(currentResultAndclickedButton));
```

Comment:
remove this `console.log` 
also you're dealing with string conactiation here 
we are adding a string to a string 0 + 1 = 01 not 1 
also 2 + 1 = 21 not 3 

so there's no point to use `parseInt` here
this is completely useles